### PR TITLE
cgroup for eve-user-apps

### PR DESCRIPTION
**Pull request by giggsoff**
We need to set memory limits for eve apps

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup [hunk 1]
```diff
@@ -111,3 +111,11 @@ for srv in $EVESRVICES; do
         /bin/echo "0-$eve_cgroup_cpus_limit" > /sys/fs/cgroup/cpuset/eve/services/"${srv}"/cpuset.cpus
     fi
 done
+
+#Creating cgroup for eve-user-apps
+mkdir -p /sys/fs/cgroup/memory/eve-user-apps
```
Comment (rvs) [inline pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup:116]: Why is this needed if all eve-user-apps are always started by containerd and we have constraints enforced from that side?

**Review by rvs — CHANGES_REQUESTED** [review]
Not sure I follow why is this needed given our containerd side of things -- please explain @giggsoff what are you trying to solve here.

File: pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup (hunk 1 above)
Comment (giggsoff) [reply pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup:116]: You mean constraints per app?
We have no limit for the set of all apps, so we faced with OOM killer for zedbox, not for apps.

Comment (rvs) [reply pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup:116]: I don't understand -- we have limits for each app. The only issue I could see if those don't add up to the # you want them to add up to. That's actually a much bigger issue with resource enforcement (currently controllers don't track wether you have enough resources for everything you specify in the config).

IOW, what you're doing here could be done much simpler if EVE start checking that all resources add up to the total resources available on the system.
